<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1029</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTabWidget" name="tab_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_tree_info">
       <attribute name="title">
        <string>tree_info</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0,0,0,0,0,0">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="path_text"/>
          </item>
          <item>
           <widget class="QPushButton" name="path_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icons/open_folder.svg</normalon>
             </iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Depth:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="depth">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="go_button">
            <property name="text">
             <string>Go</string>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icons/script.png</normalon>
             </iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="output_box">
          <property name="font">
           <font>
            <family>Courier</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_clp">
       <attribute name="title">
        <string>clp</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Not available yet</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1029</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuScript">
    <property name="title">
     <string>Script</string>
    </property>
    <addaction name="action_script_tree_info"/>
    <addaction name="action_script_clp"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_aboutQt"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScript"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_quit"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
   <addaction name="separator"/>
   <addaction name="action_about"/>
   <addaction name="action_aboutQt"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/quit.png</normaloff>:/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit the program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_script_tree_info">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/script.png</normaloff>:/icons/script.png</iconset>
   </property>
   <property name="text">
    <string>tree_info</string>
   </property>
   <property name="toolTip">
    <string>tree_info</string>
   </property>
   <property name="statusTip">
    <string>Show information about size, files and subfolders of selected folder</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>Information about author and program</string>
   </property>
  </action>
  <action name="action_aboutQt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/aboutQt.png</normaloff>:/icons/aboutQt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
   <property name="statusTip">
    <string>Information about Qt version used</string>
   </property>
  </action>
  <action name="action_script_clp">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/script.png</normaloff>:/icons/script.png</iconset>
   </property>
   <property name="text">
    <string>clp</string>
   </property>
   <property name="toolTip">
    <string>clp</string>
   </property>
   <property name="statusTip">
    <string>Transform clipboard</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/view-zoom-in.svg</normaloff>:/icons/view-zoom-in.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="statusTip">
    <string>Make the text larger</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/view-zoom-out.svg</normaloff>:/icons/view-zoom-out.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="statusTip">
    <string>Make the text smaller</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
